src/instance_check.jl

```diff
@@ -37,32 +39,32 @@ const HAS_MULTIPLE_VEHICLE_CAPACITIES = 228
 const HAS_MULTIPLE_VEHICLE_PROPERTIES = 229
 const HAS_MULTIPLE_COMPARTMENT_CAPACITIES = 230 #
 const HAS_MULTIPLE_COMPARTMENT_PROPERTIES = 231 #
+const HAS_ENERGY_RECHARGING_ISSUES_FOR_VEHICLES = 232 #
```

Comment:
some energy features such as the function type are related to locations while others are related to the vehicle. as we don't detail all the energy features now i prefer to just keep it grouped in Instance features